static/app/components/teamSelector.tsx

```diff
@@ -111,6 +111,7 @@ type Props = {
    * Controls whether the dropdown allows to create a new team
    */
   allowCreate?: boolean;
+  defaultUseIfOnlyOne?: boolean;
```

Comment:
What does this property mean? Could you add a docs comment to this?

Reply:
Yeah I can add some docs! For the invite modal, we want to default using the team if there is only 1 team option available (based on the filtering properties), so to prevent this behavior, it is a flag that controls the behavior for that use case